components/DataSourcesSection/DataSourcesSection.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { FunctionComponent } from 'react';
+import { PageSection, PageSectionHeading } from '../PageSection';
+
+interface DataSourcesSectionProps {
+  description: string;
+  methodologyUrlText: string;
+  datasourcesUrlText: string;
+  methodologyUrl: string;
+  datasourcesUrl: string;
```

Comment:
All optional.

Reply:
Following the requested changes in the CMS, this shall be taking in a description, and an array of links.

Reply:
Updated to receive an array of links